.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "astropy_helpers"]
 	path = astropy_helpers
 	url = https://github.com/Cadair/astropy-helpers.git
+        ignore = dirty
```

Comment:
Do we want to do this.

Reply:
Looks dirty... what does that suppose to do?

Reply:
You wanted it. #1858

Reply:
Wow!!! Did I write that? Dirty it is then :)

Reply:
@Cadair this ok with you?

Reply:
I am not really sure I want it, but I see the appeal. I am concerned about the interactions with `setup.py` and also if we are going to have it, it needs to be documented in the dev docs.

I think it should probably be a separate PR.